var/spack/repos/builtin/packages/silo/package.py

```diff
@@ -129,6 +130,8 @@ def flag_handler(self, name, flags):
             if spec.satisfies("%oneapi"):
                 flags.append("-Wno-error=int")
                 flags.append("-Wno-error=int-conversion")
+            if "+python" in spec:
```

Comment:
```suggestion
            if spec.satisfies("+python"):
```
(Fewer instantiations.)